src/versions/v3.ts

```diff
@@ -0,0 +1,103 @@
+// @ts-check
+import * as path from 'path'
```

Comment:
Not necessary, but just noticed it:
```suggestion
import * as path from 'node:path'
```